lib/resources/r4/types.yaml

```diff
@@ -28,7 +28,9 @@ Reference: https://hl7.org/fhir/R4/references.html
 Patient: https://hl7.org/fhir/R4/patient.html
 Practitioner: https://hl7.org/fhir/R4/practitioner.html
 Period: https://hl7.org/fhir/R4/datatypes.html#Period
+performerFunction: https://hl7.org/fhir/R4/extension-event-performerfunction.html?
```

Comment:
is that an extra `?` in the end?

Reply:
removed : https://github.com/cerner/fhir.cerner.com/commit/034df7e326ddbeda3b8fed3d80802b5e3a36b1ab#diff-b3c25dd9e50fb8a137ff641afe13c3b7L31